modules/claims_api/app/swagger/claims_api/form_0966_v1_controller_swagger.rb

```diff
@@ -3,5 +3,235 @@
 module ClaimsApi
   class Form0966V1ControllerSwagger
     include Swagger::Blocks
+
+    swagger_path '/form/0966' do
+      operation :post do
+        key :summary, 'Accepts 0966 Intent to File form submission'
+        key :description, 'Accepts JSON payload. Full URL, including\nquery parameters.'
+        key :operationId, 'post0966itf'
+        key :tags, [
+          'Intent to File'
+        ]
+
+        parameter do
+          key :name, 'bearer_token'
```

Comment:
Is this accurate? Should it not be `apikey` too?

Reply:
V1 is bearer token based and opened like veteran verification, v0 is apikey based.